packages/elements/src/slider/utils.ts

```diff
@@ -39,22 +39,23 @@ const countDecimalPlace = (value: string | number): number => {
 };
 
 /**
- * Check if the value is a valid number and convert to string
+ * Check if the value is a valid number
  * If not, return fallback value instead
  * @param value value for checking
  * @param fallbackValue fallback value when value is not number;
  * @returns string of valid number
  */
-const validateNumber = (value: number, fallbackValue: number | string): string => {
+const validateNumber = (value: number, fallbackValue: number): number => {
```

Comment:
based on function name, I would expect it to return `NaN` if passed value is invalid number or not a number. The more correct would be:
```
const toNumber = (value: unknown, fallbackValue = 0): number => {
   if (typeof value === 'number' && !isNaN(value)) {
     return value;
   }
   return fallbackValue;
}
```

Reply:
Actually we may not need this function at all. All the getters already provided fallback values. Please see https://github.com/Refinitiv/refinitiv-ui/pull/299/commits/6633de30c30c4c28f47b171454346b40864a8450